packages/server/src/api/controllers/query/validation.ts

```diff
@@ -10,11 +10,14 @@ function baseQueryValidation() {
     fields: Joi.object().required(),
     datasourceId: Joi.string().required(),
     readable: Joi.boolean(),
-    parameters: Joi.array().items(
-      Joi.object({
-        name: Joi.string(),
-        default: Joi.string().allow(""),
-      })
+    parameters: Joi.alternatives(
```

Comment:
I think this might not be quite the right format, I think it is supposed to be:
```js
Joi.alternatives().try(
  Joi.array().items(
    Joi.object({
      name: Joi.string(),
      default: Joi.string().allow(""),
    })
  ),
  Joi.object({}).required().unknown(true)
)
```